shell/src/main/java/alluxio/cli/fs/command/LoadCommand.java

```diff
@@ -38,13 +73,13 @@
 @ThreadSafe
 @PublicApi
 public final class LoadCommand extends AbstractFileSystemCommand {
-  private static final Option LOCAL_OPTION =
-      Option.builder()
-          .longOpt("local")
-          .required(false)
-          .hasArg(false)
-          .desc("load the file to local worker.")
-          .build();
+  // todo(jianjian) add option for timeout and parallelism(compatibility?)
+  private static final int DEFAULT_PARALLELISM = 4;
```

Comment:
Are we making the `LoadCommand` concurrent? Isn't `DistributedLoad` is intended for such uses?

Reply:
done, keep it simple and run the task one by one then